polaris.shopify.com/src/components/Frame/Frame.tsx

```diff
@@ -284,6 +284,26 @@ function NavItem({
               </li>
             );
           })}
+      <li>
```

Comment:
I've added a link to the sandbox as the last top level nav item here, I don't think it belongs in the generated cache nav.json, as that's not what its designed for. 

Likely we'll want to remove this in the future in favour of either a promo on the homepage, or a more well considered CTA cc @martenbjork